evap/rewards/tests/test_views.py

```diff
@@ -205,21 +205,26 @@ def setUpTestData(cls):
             course=course,
         )
 
-        cls.url = f"/rewards/reward_semester_activation/{cls.semester.pk}/"
+        cls.url = reverse("rewards:semester_activation_edit", args=[cls.semester.pk])
+
+    def test_invalid(self):
```

Comment:
In theory, we could also do one case that checks what happens if the parameter is missing